### PR TITLE
[BEAM-157] Fix erroneous test in CombineTest

### PR DESCRIPTION
**Pull request by kennknowles**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by kennknowles** [comment]
R: @bjchambers

**Comment by kennknowles** [comment]
The two commits are independent and should not be squashed. The first is just automated style polish.

File: sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineTest.java [hunk 1]
```diff
@@ -389,16 +391,26 @@ public void testGlobalCombineWithDefaultsAndTriggers() {
         .apply(Sum.integersGlobally())
         .apply(ParDo.of(new FormatPaneInfo()));
 
-    PAssert.that(output).containsInAnyOrder("1: false", "2: true");
+    // The elements produced are nondeterministic, but it should certainly have
```
Comment (bjchambers) [inline sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineTest.java]: Emphasize that whether or not we *actually* get the partial result is non deterministic. At first I thought you just meant the order, so I was going to suggest the in any order again ;)

Comment (kennknowles) [inline sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineTest.java]: Done.

**Comment by bjchambers** [comment]
LGTM

**Pull request closed without merging**